ReactAndroid/src/main/java/com/facebook/react/uimanager/events/EventDispatcher.java

```diff
@@ -129,6 +130,24 @@ public void dispatchEvent(Event event) {
       // touch event dispatch will hit this codepath, and we simply queue them so that they
       // are dispatched once ReactContext creation completes and JS app is running.
     }
+
+    for (EventDispatcherListener listener : mListeners) {
+      listener.onEventDispatched(event);
```

Comment:
Here was my response on the old diff:

I think we should still send events to JS since some component could do some JS logic in their callback and there is also the event listener that we would still need to call into js for https://github.com/facebook/react-native/blob/master/Libraries/Animated/src/AnimatedImplementation.js#L1887